src/Authenticator/LoginHandler.php

```diff
@@ -58,7 +55,8 @@ public function doLogin($data, MemberLoginForm $form, HTTPRequest $request)
         $member = $this->checkLogin($data, $request, $result);
 
         // If there's no member it's an invalid login. We'll delegate this to the parent
-        if (!$member) {
+        // Additionally if there are no MFA methods registered then we will also delegate
+        if (!$member || !$this->getMethodRegistry()->areMethodsAvailable()) {
```

Comment:
I'm not 100% on this naming. What about `getMethods()` or `hasMethods()` or something? Something a bit more uniform in the whole get/set/is/has area would be nice

Reply:
Well I use `is` for booleans, but then `are` for `is` on a plural 😅 . `hasAvailableMethods` is nice though - I'll use that.

Reply:
I guess `hasMethods` is also good - available is implied.